docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md

```diff
@@ -231,18 +231,19 @@ YugabyteDB Anywhere will end support for Replicated installation at the end of 2
 
 If your YBA installation uses Replicated, you can use YBA Installer to migrate from Replicated.
 
-Review the [prerequisites](../../prerequisites/installer/). YBA Installer performs the migration in place. Make sure you have enough disk space on your current machine.
+### Before you begin
```

Comment:
@ddhodge - Maybe you can copy these steps to your Prerequisites reorganization PR - https://github.com/yugabyte/yugabyte-db/pull/21876/files

Reply:
@ddhodge @aishwarya24 : That works, let me know if I can abandon this PR